### DISCUSSION TITLE
上下文的联系能力怎么样？

### DISCUSSION THREAD

**Discussion by Li1177** [ChatGPTNextWeb/NextChat#727] [Q&A]
从来不写代码的，完全不懂的。有了gpt以后，发现真的可以尝试很多。
我想问下你的这个程序调用3.5的时候，上下文联系功能怎么样？
我在网上找了很多python的代码，少数是可以支持调用3.5的，而且
是有一个是可以有上下文功能的。但是在本机上使用，速度很慢。
输入框输入以后，一点send就超级卡顿。

**Comment by lkbgct** [comment]
今天我重新部署了下，觉着上下文能力很棒了，只是不知是在线时间长还是其他原因，会出错，刷新下网页就好了，不会丢失聊天记录。
